command/perf_send.go

```diff
@@ -0,0 +1,104 @@
+package command
+
+import (
+	"context"
+	"path/filepath"
+
+	"github.com/evergreen-ci/evergreen/model"
+	"github.com/evergreen-ci/evergreen/rest/client"
+	"github.com/evergreen-ci/poplar"
+	"github.com/evergreen-ci/poplar/rpc"
+	"github.com/mitchellh/mapstructure"
+	"github.com/pkg/errors"
+)
+
+type perfSend struct {
+	// AwsKey and AwsSecret are the user's credentials for authenticating
+	// interactions with s3. These are required if any of the tests have
+	// artifacts.
+	AWSKey    string `mapstructure:"aws_key" plugin:"expand"`
+	AWSSecret string `mapstructure:"aws_secret" plugin:"expand"`
+
+	// Region is the s3 region where the global bucket is located. It
+	// defaults to "us-east-1".
+	Region string `mapstructure:"region" plugin:"expand"`
+
+	// Bucket is the global s3 bucket to use when storing any artifacts
+	// without a bucket specified.
+	Bucket string `mapstructure:"bucket" plugin:"expand"`
+
+	// Prefix specifies the global prefix to use within the s3 bucket for
+	// any artifacts without a prefix specified.
+	Prefix string `mapstructure:"region" plugin:"expand"`
+
+	// File is the file containing either the json or yaml representation
+	// of the performance report tests.
+	File string `mapstructure:"file" plugin:"expand"`
```

Comment:
Does it have to be all in one file? Do they have the option of uploading multiple files, or would this never happen?

Reply:
It is all in one file, not possible to have multiple files.